staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -212,3 +228,109 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 
 	return s, nil
 }
+
+func defaultAPIResourceConfigSource() *serverstorage.ResourceConfig {
```

Comment:
`defaultAPIResourceConfigSource` and `mergeAPIResourceConfigs` are exactly the same with those in `staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go`.

Better to move to `staing/src/k8s.io/apiserver/pkg/util/<xx-folder>`. May create a new folder if needed.

Reply:
yes,  make sense. 
 @sttts, what's your opinion?

Reply:
yes, they should be at a central place in k8s.io/apiserver. pkg/util or even pkg/server/resourceconfig or something like that?

Reply:
But have one issue, if move to `staing/src/k8s.io/apiserver/pkg/util/<xx-folder>`, the pkg will import `	serverstorage "k8s.io/apiserver/pkg/server/storage"
`
this may be not acceptable.

Reply:
why not into `k8s.io/apiserver/pkg/server/resourceconfig`?

Reply:
SGTM